lib/ansible/modules/windows/win_uri.ps1

```diff
@@ -232,8 +234,12 @@ if ($return_content -or $dest) {
             $memory_st.Seek(0, [System.IO.SeekOrigin]::Begin)
             $content_bytes = $memory_st.ToArray()
             $result.content = [System.Text.Encoding]::UTF8.GetString($content_bytes)
-            if ($result.ContainsKey("content_type") -and $result.content_type -in @("application/json", "application/javascript")) {
-                $result.json = ConvertFrom-Json -InputObject $result.content
+            if ($result.ContainsKey("content_type") -and $result.content_type -Match ($JSON_CANDIDATES -join '|')) {
+                try {
+                    $result.json = ConvertFrom-Json -InputObject $result.content
+                } finally {  # We use 'finally' to avoid PSAvoidUsingEmptyCatchBlock issues
```

Comment:
This doesn't do what you think, without the catch block then any terminating errors will not be caught and error out. The finally block only means the code in that block will run at the end regardless of an error or not.

In this case I could do

```
try {
    $result.json = ConvertFrom-Json -InputObject $result.content
} catch [System.ArgumentException] {}
```

The `PSAvoidUsingEmptyCatchBlock` will still fire but this sounds like a case where we can ignore it for this scenario.

Reply:
Yup, it was a test I did this morning to see if I could avoid the PSAvoidUsingEmptyCatchBlock which I think is a stupid oversimplification of programming.

Reply:
I removed the test-commit now.